generator/internal/metadata_decorator_generator.cc

```diff
@@ -397,16 +397,21 @@ std::unique_ptr<::google::cloud::internal::AsyncStreamingReadRpc<
       continue;
     }
     if (IsStreamingWrite(method)) {
+      // Asynchronous streaming writes do not consume a request, typically the
+      // first `Write()` call contains any relevant data to "start" the stream.
+      // Thus, the decorator cannot add any routing instructions. The caller
+      // should initialize `context` with any such instructions. At this time,
```

Comment:
I think I would just say: "The handwritten caller should initialize \`context\` with any such instructions." and delete the rest of the paragraph.

If you decide to keep the rest of the paragraph then, `s/, ut/, but/` a few lines down.

Reply:
Option 1